.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,15 +1,20 @@
 ---
 name: Bug report
 about: Create a report to help reproduce and fix the issue
-
 ---
+<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable -->
+## Round ID:
+
+<!--- **INCLUDE THE ROUND ID**
+If you discovered this issue from playing tgstation hosted servers:
+[Round ID]: # (It can be found in the Status panel or retrieved from https://atlantaned.space/statbus/round.php ! The round id let's us look up valuable information and logs for the round the bug happened.)-->
+
+## Testmerges:
 
-[Round ID]: # (If you discovered this issue from playing tgstation hosted servers:)
-[Round ID]: # (**INCLUDE THE ROUND ID**)
-[Round ID]: # (It can be found in the Status panel or retrieved from https://atlantaned.space/statbus/round.php ! The round id let's us look up valuable information and logs for the round the bug happened.)
+<!-- If you believe the issue to be caused by a test merge [OOC tab -> Show Server Revision], report it in the pull request's comment section instead. If None, feel free to remove this section. -->
```

Comment:
I would change this to remove the "if you believe part", as some testmerges might mess up seemingly unrelated functionality, and as such should always be listed